src/app/pipes/valuestyle.pipe.ts

```diff
@@ -36,7 +36,7 @@ export class ValuestylePipe implements PipeTransform {
 		if (args.length == 2) {
 			const secondDrop = args[1] instanceof BigNumber ? args[1] : new BigNumber(args[1])
 			if (value.isGreaterThanOrEqualTo(firstDrop)) return POSITIVE
-			else if (value.isLessThanOrEqualTo(secondDrop)) return WARNING
+			else if (value.isGreaterThanOrEqualTo(secondDrop)) return WARNING
```

Comment:
Here is how I think these parameters should work:
```
100 - 90% is good
89 - 60% is average
59 - 0% is bad
=>
First Drop: 90
Second Drop: 60
```

Please note that this pipe using two parameters is already used here:
https://github.com/gobitfly/eth2-beaconchain-explorer-app/blob/b55781bcc668a39f525d10ae14ba6a3ea5d7d2a1/src/app/components/dashboard/dashboard.component.html#L531

Reply:
Ok

Reply:
lgtm, seems more intuitive and since we have value that uses the pipe with 2 arguments we can change it.